changelog/v1.18.0-beta35/bump-kubectl-image.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
nit: should this be user-facing? maybe FIX

Reply:
I wasn't sure if this was actually a user facing image or just something used in our tests - I couldn't find references anywhere to it in this repo outside of tests.

In GME this image is conditionally used for an upgrade job related to automating migration of breaking prometheus changes, so I'll have a user-facing changelog there when I bump it in that repo.

Reply:
we actually use this image in some jobs run during install/upgrade - see [rollout job](https://github.com/solo-io/gloo/blob/main/install/helm/gloo/templates/5-resource-rollout-job.yaml) and where it gets its [image value](https://github.com/solo-io/gloo/blob/833eb9d7d1814630f3d11ff2173a1028de829529/install/helm/gloo/values-template.yaml#L171) from (so it's not used for testing only)

Reply:
Great, I'm sold, will change to `FIX` then so it's user facing.